src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py

```diff
@@ -1583,6 +1586,7 @@ def quantize_torch_qat_export(
     _convert_quantizable_gemm_no_activations(model)
     quantize_resnet_identity_add_inputs(model)
     _remove_duplicate_quantize_ops(model)
+    _convert_signed_to_unsigned(model)
```

Comment:
Is there a reason behind shifting this signed to unsigned conversion at the last?